src/transformStateWithClones.js

```diff
@@ -7,7 +7,32 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let myState = { ...state };
+  const result = [];
+
+  for (const obj of actions) {
```

Comment:
```suggestion
  for (const action of actions) {
```

We have array of actions so each element of this array in one action